dipy/viz/actor.py

```diff
@@ -201,6 +201,142 @@ def copy(self):
     return image_actor
 
 
+def contour_actor(data, affine=None, levels=[50],
+                  colors=[np.array([1.0, 0.0, 0.0])], opacities=[1]):
+    """Take a volume and draw surface contours for any any number of
```

Comment:
Hi @kesshijordan, thank you for your PR. Can you add a small tutorial in docs/example? @ranveeraggarwal, @MarcCote  can you explain to @kesshijordan how to write tests for this PR?